doc/source/whatsnew/v0.19.1.txt

```diff
@@ -44,4 +44,5 @@ Bug Fixes
 
 
 - Bug in ``pd.concat`` where names of the ``keys`` were not propagated to the resulting ``MultiIndex`` (:issue:`14252`)
+- Bug in ``pd.concat`` where ``axis`` cannot take string parameters ``rows`` or ``columns (:issue:`14369`)
```

Comment:
missing closing backticks after "columns" 
Small other thing: I would also quote the parameters inside the backticks (like ``` ``'rows' `` ```) to make clear it are string arguments